### PR TITLE
Form

### PR DESCRIPTION
**Pull request by phelian23**
Add a contact form to the mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -199,14 +199,15 @@ <h2>About me</h2>
         </section>
         <section class="form">
             <div class="formflex">
-                <p>I'm always interested<br>in hearing about new<br>projects, so if you'd<br>like to chat please<br>get in touch.</p>
+                <h2>I'm always interested<br>in hearing about new<br>projects, so if you'd<br>like to chat please<br>get in touch.</h2>
             <form>
```
Comment (TedLivist) [inline index.html:203]: Your form should submit properly using the [Formspree service](https://formspree.io/html) as required in the description.
- [ ] The action value should be similar to https://formspree.io/f/{form_id}
- [ ] method value equals POST

Comment (TedLivist) [inline index.html]: Your form should submit using the [Formspree service](https://formspree.io/html) as required in the description.
- [ ] `action` value should be similar to `https://formspree.io/f/{form_id}`
- [ ] `method` value equals `POST`

File: index.html [hunk 2]
```diff
@@ -199,14 +199,15 @@ <h2>About me</h2>
...
-                <input class="name1" type="text" placeholder="Full Name">
+                <input class="name1" type="text" placeholder="Full Name" maxlength="30">
                 <input class="name" type="text" placeholder="First Name">
                 <input class="name" type="text" placeholder="Last Name"><br>
-                <input class="email" type="text" placeholder="Email address"><br>
-                <input class="note" type="text" placeholder="Hello Mariane. Glad to see you!"><br>
-                <button type="submit">Get in touch</button>
+                <input class="email" type="email" placeholder="Email address"><br>
+                <textarea class="note" placeholder="Hello Femi. Glad to see you!" maxlength="500"></textarea><br>
```
Comment (TedLivist) [inline index.html]: - [ ] Please all inputs should have the `required` attribute to activate the validation on submit.

File: index.html [hunk 3]
```diff
@@ -199,14 +199,15 @@ <h2>About me</h2>
...
+                <div class="btn"><button type="submit">Get in touch</button></div>
+
```
Comment (TedLivist) [inline index.html]: - [ ] Please remove the blank line as it is unnecessary

blank_m

File: index.html [hunk 4]
```diff
@@ -199,14 +199,15 @@ <h2>About me</h2>
...
                 <input class="name" type="text" placeholder="Last Name"><br>
-                <input class="email" type="text" placeholder="Email address"><br>
-                <input class="note" type="text" placeholder="Hello Mariane. Glad to see you!"><br>
-                <button type="submit">Get in touch</button>
+                <input class="email" type="email" placeholder="Email address"><br>
+                <textarea class="note" placeholder="Hello Femi. Glad to see you!" maxlength="500"></textarea><br>
+                <div class="btn"><button type="submit">Get in touch</button></div>
+                
             </form>
             </div>
```
Comment (TedLivist) [inline index.html]: - [ ] Please fix your indentation. `<form>` element should be indented.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️ 

Hi @phelian23 and @Chimwemwe-127001,

Good job so far!
- You have done great with the design
- Your linters are working properly.
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ]  Please fix your background color. It is different from what is on the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

**Figma template:**
phels-orig

**Your design:**
phels-new

- [ ] **_Check the comments under the review._**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by phelian23** [comment]
Thanks for the review. for the color difference in the form section, I don't know what's the cause, because I used the exact color code in the template

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🎉**
Hello @phelian23 👋🏽
Your project is complete!  There is nothing else to say. It's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**